lib/core/src/model.rs

```diff
@@ -81,9 +112,12 @@ pub struct Config {
 impl Config {
     pub fn mainnet(breez_api_key: Option<String>) -> Self {
         Config {
-            liquid_electrum_url: "elements-mainnet.breez.technology:50002".to_string(),
-            bitcoin_electrum_url: "bitcoin-mainnet.blockstream.info:50002".to_string(),
-            mempoolspace_url: "https://mempool.space/api".to_string(),
```

Comment:
Since we don't depend on mempool anymore, we can get rid of the mempool instance on the regtest setup. It's best to rebase first (there are regtest changes to be taken in).